CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
+Added:
+  * (CPR-753) Changed permissions for puppetserver logdir from 700 to 750.
```

Comment:
Should be reworded to not call out puppetserver since this will affect all the logdirs for all the projects.